app/controllers/check-and-send.ts

```diff
@@ -0,0 +1,75 @@
+import { NextFunction, Request, Response, Router } from 'express';
+import moment from 'moment';
+import { createDummyAppealApplication } from '../data/dummy';
+import { paths } from '../paths';
+import { statementOfTruthValidation } from '../utils/fields-validations';
+import { addSummaryRow, Delimiter } from '../utils/summary-list';
+
+function createSummaryRowsFrom(appealApplication: AppealApplication) {
```

Comment:
I would probably in this case use the HMTL version for the summary instead of the Nunjucks component, so we would have a check your answers page per journey but this would work as well so I leave it to you to decide